internal/ethapi/api.go

```diff
@@ -927,10 +927,13 @@ func (e *revertError) ErrorData() interface{} {
 //
 // Note, this function doesn't make and changes in the state/blockchain and is
 // useful to execute and retrieve values.
-func (s *PublicBlockChainAPI) Call(ctx context.Context, args CallArgs, blockNrOrHash rpc.BlockNumberOrHash, overrides *map[common.Address]account) (hexutil.Bytes, error) {
-	var accounts map[common.Address]account
+func (s *PublicBlockChainAPI) Call(
+	ctx context.Context, args CallArgs,
+	blockNrOrHash rpc.BlockNumberOrHash, overrides map[common.Address]Account,
```

Comment:
The point of `*map[common.Address]account` is to differentiate between mandatory an optional arguments in the RPC layer. Please revert.

Reply:
but if caller doesn't provide the map - anyway it will be nil, no?